app/poros/search_results.rb

```diff
@@ -1,22 +1,30 @@
 class SearchResults
 
-  def repos(token)
-    json = GithubService.new.list_repos(token)
-    @repos = json[0..4].map do |user_data|
+  def initialize(token)
```

Comment:
This whole file has been refactored to take in a token from controller --> github service

Reply:
Thank you for the break down, makes this a lot easier to understand!